bin/webcm.js

```diff
@@ -96,8 +103,13 @@ function onFatalError(error) {
           transpileOnly: true,
           dir: __dirname,
         })
-        const { startServer } = require('../lib/server')
-        startServer(argv.config, argv.components)
+        const { startServerFromConfig } = require('../lib/server')
+        startServerFromConfig({
```

Comment:
did we do this thing that allows every config setting to be set using a CLI argument, or did we give up on it for now?